source/Target/Target.cpp

```diff
@@ -2494,6 +2494,15 @@ SwiftASTContextReader Target::GetScratchSwiftASTContext(
         llvm::cast_or_null<SwiftASTContext>(GetScratchTypeSystemForLanguage(
             &error, eLanguageTypeSwift, create_on_demand));
   }
+
+  StackFrameWP frame = exe_scope.CalculateStackFrame();
+  StackFrameSP frame_sp = StackFrameSP(frame.lock());
```

Comment:
Does this work when you do:
```
lldb a.out
(lldb) p global_var
```

without running the program first?

Reply:
Checked with Adrian this doesn't regress what's already there.